assets/src/components/EventItem.js

```diff
@@ -19,6 +19,8 @@ const EventItem = ( props ) => {
 
 	const eventClass = `gp-events-list`;
 
+	const venue = event.venue;
```

Comment:
what do you think of something like...
``const venue = <a href="`${ venue.permalink }`">`${ venue.name }`</a>``